src/components/settings/SettingsEditorTab.vue

```diff
@@ -67,6 +76,25 @@ export default class SettingsEditorTab extends Mixins(BaseMixin) {
         },
     ]
 
+    private tabSizes = [
```

Comment:
i would use here a getter method with a loop. pls also use i18n for the word "Spaces" here.

Reply:
Good idea, I have made a new commit with the suggested changes